core/store/src/columns.rs

```diff
@@ -283,6 +282,21 @@ const OPTIONAL_GC_COLUMNS: [bool; DBCol::COUNT] = col_set(&[
     DBCol::ColStateParts,
 ]);
 
+// The columns below are 'reference-counted'. This means, that we're storing additional
+// 8 bytes at the end of the payload with the current RC value.
```

Comment:
Could you add a bit on the semantics of negative/zero RC counts? I remeber being puzzled a lot by this function:

```rust
/// Returns
/// (Some(value), rc) if rc > 0
/// (None, rc) if rc <= 0
pub fn decode_value_with_rc(bytes: &[u8]) -> (Option<&[u8]>, i64) {
```